tests/migration/Core/V6_5/Migration1663402950SetDoubleOptinCustomerActiveTest.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_5;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Test\TestCaseBase\BasicTestDataBehaviour;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\Migration\V6_5\Migration1663402950SetDoubleOptinCustomerActive;
+
+/**
+ * @internal
+ *
+ * @covers \Shopware\Core\Migration\V6_5\Migration1663402950SetDoubleOptinCustomerActive
+ */
+class Migration1663402950SetDoubleOptinCustomerActiveTest extends TestCase
+{
+    use KernelTestBehaviour;
```

Comment:
You should not use the KernelTestBehaviour anymore, as we don't want to boot the complete kernel for the migration tests anymore, furthermore i don't see where you actually use the `BasicTestDataBehaviour` in this test either, it should be enough to use the `MigrationTestTrait`, to ensure that the transaction is rollbacked after each test execution, to ensure that your test does not change the state of the database